website/api/generating-proofs/local-proving.md

```diff
@@ -6,14 +6,82 @@ The [feature flags] include options to toggle between CPU and GPU proving.
 
 ## Local vs. Remote Proving
 
-For most use cases, we recommend remote proving using [Bonsai].
-The primary reason to consider using local proof generation for your application is to manage private data.
+For most use cases, we recommend remote proving using [Bonsai], our scalable and highly parralel proving service.
 
-> Whoever is generating the proofs can see all private information involved.
+Proving with **private data** is a key reason to consider using local proof generation, as whoever is generating the proofs can see all private information involved.
+With local proof generation your private data never leaves your machine.
 
-_With local proof generation, you can generate proofs about your private data, while your private data never leaves your machine._
+## Proving hardware
+
+RISC Zero's proving system for multiple hardware targets.
+
+:::tip
+In cases where memory is constrained (i.e. less than 10 GB is availble), it may be necessary to change the [segment size limit][segment-limit-docs-latest].
+You can find information about expected memory consumption on our [benchmarks page][datasheet].
+
+:::
+
+:::info
+The Groth16 prover currently _only_ works on x86 architecture, and so Apple Silicon is _currently unsupported_ (even via Docker).
+
+You can find out more info in the relevant issues [here](https://github.com/risc0/risc0/issues/1520) and [here](https://github.com/risc0/risc0/issues/1749).
+:::
+
+### CPU
+
+RISC Zero proving will run on nearly any modern CPU (x86 or ARM).
+Proving on CPU is the easiest to setup, and the most portable, but will not result in the highest performance or compute costs.
+
+On platforms other than MacOS, CPU proving unless otherwise configured.
```

Comment:
```suggestion
On platforms other than MacOS, CPU proving is the default unless otherwise configured.
```

Note that this will change soon. So we could just leave this out.

Reply:
Took it out 👍 
7d206eae4b66aaeba66d1b76b0fa3d51c3132b66